modules/visual_script/visual_script_builtin_funcs.h

```diff
@@ -72,6 +72,7 @@ class VisualScriptBuiltinFunc : public VisualScriptNode {
 		MATH_RANDI,
 		MATH_RANDF,
 		MATH_RANDF_RANGE,
+		MATH_RANDFN,
 		MATH_RANDI_RANGE,
```

Comment:
The order is a bit awkward, I would change it to match `variant_utility`:
```suggestion
		MATH_RANDI,
		MATH_RANDF,
		MATH_RANDFN,
		MATH_RANDI_RANGE,
		MATH_RANDF_RANGE,
```
And thus moving stuff around elsewhere to match that order.

Reply:
Note that my suggestion also swaps `RANDI_RANGE` and `RANDF_RANGE` which were not in the same order as `RANDI` and `RANDF`.

Could also be this everywhere:
```
randi
randf
randi_range
randf_range
randfn
```

Reply:
Done (I think)